text/0010-dependency-mappings.md

```diff
@@ -11,10 +11,26 @@ Consumers of Paketo buildpacks may be building in an environment where certain d
 ## Detailed Explanation
 
 ### Discovery
-If one or more bindings of kind or type `dependency-mapping` are present, buildpacks should respect any URIs found in the binding. The `sha256` of the dependency should be used to lookup mapped URIs.
+If one or more bindings of kind or type `dependency-mapping` are present, buildpacks should respect any URIs found in the binding. The lookup digest should be included as `checksum` in the form of `<algorithm>:<hash>` (or `sha256` in the form of `<hash>`) of the dependency should be used to lookup mapped URIs.
```

Comment:
```suggestion
If one or more bindings of kind or type `dependency-mapping` are present, then a buildpack should use the bindings in its dependency resolution process. The dependency's digest, included as `checksum` in the form of `<algorithm>:<hash>` (or `sha256` in the form of `<hash>`), should be used to lookup an alternative URIs.
```

Splitting hairs, but I think this sentence is a little confusing. Suggestion ^^.

Reply:
made a slight change, incorporating it now.